api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -67,6 +71,15 @@ public void setUserDAO(UserDAO dao) {
 		this.dao = dao;
 	}
 	
+	/**
+	 * Sets the time in seconds for password reset activation key to be valid
+	 * @param validTime the validTime to set
+	 */
+	public void setValidTime(int validTime) {
```

Comment:
I die mention that this method should not be public

Reply:
What is the use of this method? You should have a utility method that returns the valid time, i'm it should be something like getValidTime()